packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -327,4 +332,20 @@ export class GutenbergEditorPage {
 		await Promise.all( [ this.page.waitForNavigation(), frame.click( selectors.viewButton ) ] );
 		await this.page.waitForLoadState( 'networkidle' );
 	}
+
+	/**
+	 * Opens the Block Editor sidebar (left hand side).
+	 */
+	async toggleSidebar(): Promise< void > {
```

Comment:
Can we give this method a slightly more descriptive name to differentiate it from the settings sidebar? Just that sidebar is an overloaded term with the settings sidebar, and I know sometimes folks just go off of method name when working quickly.

Reply:
Yeah, you're right - the DOM references this element as `navsidebar`, so how about that?

Reply:
Updated to `openNavSidebar` in bd8bc48bb555cb23660d753d3d43076801065851.